Cat2Cat/main.m

```diff
@@ -207,10 +218,34 @@ int main(int argc, const char * argv[])
         [assetCatalogPaths addObjectsFromArray:filesMatchingPattern(url.path)];
     }
     parameters.assetCatalogPaths = assetCatalogPaths;
-    
+
+    parameters.methodNamePrefix = methodNamePrefix;
+
     NSURL *outputUrl = [NSURL URLWithString:[outputDirectory stringByAddingPercentEscapesUsingEncoding:NSUTF8StringEncoding]
                               relativeToURL:basePathUrl];
     parameters.outputDirectory = outputUrl.path;
     
     return parameters;
 }
+
+static BOOL validateMethodNamePrefix(NSString *methodNamePrefix, NSError **error)
+{
+    if ([methodNamePrefix length] < 2) {
+        *error = [NSError errorWithDomain:@"Cat2Cat" code:-1 userInfo:@{ NSLocalizedDescriptionKey : @"Method name prefix must be at least 2 characters." }];
```

Comment:
Yeah, reverse DNS is indeed the way. `io.vokal.Cat2Cat` because we're paying the indian ocean territories for our website.